classes/rust.bbclass

```diff
@@ -1,7 +1,6 @@
 RUSTC = "rustc"
 
-# FIXME: --sysroot might be needed
-RUSTC_ARCHFLAGS += "--target=${TARGET_SYS} -C rpath"
+RUSTC_ARCHFLAGS += "--target=${TARGET_SYS} --sysroot=${STAGING_DIR_NATIVE}/${prefix_native} -C rpath"
```

Comment:
Again, is this sysroot always correct?

What type of failure did not having it trigger?